code/modules/mob/living/simple_animal/friendly/diona.dm

```diff
@@ -42,103 +43,99 @@
 	holder_type = /obj/item/holder/diona
 	can_collar = 1
 
+	a_intent = INTENT_HELP
+	var/gestalt_alert = "merged with gestalt"
+	var/evolve_donors = 5
+	var/awareness_donors = 3
+	var/nutrition_need = 500
+	
+
 /mob/living/simple_animal/diona/New()
 	..()
 	if(name == initial(name)) //To stop Pun-Pun becoming generic.
 		name = "[name] ([rand(1, 1000)])"
 		real_name = name
 
 	add_language("Rootspeak")
-	verbs += /mob/living/simple_animal/diona/proc/merge
+
+/mob/living/simple_animal/diona/UnarmedAttack(var/atom/A)
+	if(isdiona(A) && src in A.contents) //can't attack your gestalt
```

Comment:
the `src in A.contents` should be in its own set of parens, to avoid operator grouping causing errors